libs/libcommon/src/libcommon/orchestrator.py

```diff
@@ -211,14 +224,44 @@ def run(self) -> TasksStatistics:
             return TasksStatistics(num_deleted_cache_entries=delete_dataset_responses(dataset=self.dataset))
 
 
+@dataclass
+class UpdateRevisionOfDatasetCacheEntriesTask(Task):
+    dataset: str
+    old_revision: str
+    new_revision: str
+
+    def __post_init__(self) -> None:
+        # for debug and testing
+        self.id = "UpdateRevisionOfDatasetCacheEntriesTask,1"
+        self.long_id = self.id
+
+    def run(self) -> TasksStatistics:
+        """
+        Update the revision of the dataset cache entries.
+
+        Returns:
+            `TasksStatistics`: The statistics of the cache entries updates.
+        """
+        with StepProfiler(
+            method="UpdateRevisionOfDatasetCacheEntriesTask.run",
+            step="all",
+            context=f"dataset={self.dataset}",
+        ):
+            return TasksStatistics(
+                num_updated_cache_entries=update_revision_of_dataset_responses(
+                    dataset=self.dataset, old_revision=self.old_revision, new_revision=self.new_revision
+                )
+            )
+
+
 @dataclass
 class DeleteDatasetStorageTask(Task):
     dataset: str
     storage_client: StorageClient
 
     def __post_init__(self) -> None:
         # for debug and testing
-        self.id = f"DeleteDatasetStorageTask,{self.dataset},{self.storage_client}"
+        self.id = f"DeleteDatasetStorageTask,{self.dataset},1"
```

Comment:
I'm not sure: I think we want to show the details of the storage client here (see it `__repr__()` method)

Reply:
I switched to {self.storage_client.protocol}://{self.storage_client.storage_root}